playbooks/logging.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
maybe `telemetry-logging` or `enable-compute-logging` would be more appropriate for the playbook name?

`logging` seems too generic.